README.md

```diff
@@ -15,7 +15,7 @@ npm install --save string_decoder
 
 This package is a mirror of the string_decoder implementation in Node-core.
 
-Full documentation may be found on the [Node.js website](https://nodejs.org/dist/v7.8.0/docs/api/).
+Full documentation may be found on the [Node.js website](https://nodejs.org/dist/latest/docs/api/string_decoder.html).
```

Comment:
Unfortunately that is updated automatically. So this was taken from v7.8.0.
https://github.com/rvagg/string_decoder/blob/master/build/build.js#L146-L151